### PR TITLE
[Getting Started] Add a Getting Started Guide

### PR DESCRIPTION
**Pull request by anayini**
## Changes in this pull request
Add a Getting Started Guide

### CODE REVIEW

File: Guides/Getting Started.md [hunk 1]
```diff
@@ -1,3 +1,128 @@
 # Getting Started
 
-> TODO:
+ReactiveLists provides a React-like API for `UITableView` and `UICollectionView`.  This goal is to provide a more
+declarative interface on top of your normal list and collection code.  To get started with ReactiveLists, we encourage you to play around with the examples included in the repository, but here we will also discuss the main components of the library.
```
Comment (jessesquires) [inline Guides/Getting Started.md]: > on top of your normal list and collection code.

👉 

> on top of your normal **table** and collection code

Comment (jessesquires) [reply Guides/Getting Started.md]: > but here we will also discuss the main components of the library.

I think we can remove this line.

File: Guides/Getting Started.md [hunk 2]
```diff
@@ -1,3 +1,128 @@
...
+
+To get set up, you first need to add a `Driver` (either a `TableViewDriver` or `CollectionViewDriver`) to your view controller.
```
Comment (jessesquires) [inline Guides/Getting Started.md]: I think all of the explanation should come first. *then* example code.

File: Guides/Getting Started.md [hunk 3]
```diff
@@ -1,3 +1,128 @@
...
+A `CellViewModel` just needs to conform to `TableViewCellViewModel` or `CollectionViewCellViewModel`.  The
+most important function is `applyViewModelToCell`.  This is where you define how your view model will get
+mapped onto a `UITableViewCell` or `UICollectionViewCell`.  In this case we are just setting the `textLabel`'s
+`text` property to the person's `name`.
+
+And there you have it!  You now have an automatically refreshing table view which you defined in a clear
+declarative manner.
+
+
+### Glossary of types
```
Comment (jessesquires) [inline Guides/Getting Started.md]: RE: comment above

I think this should read "Primary Components", then have the explanations below. 😊

File: README.md [hunk 4]
```diff
@@ -14,6 +14,36 @@ For long-term goals and direction, please see [`VISION.md`](https://github.com/p
 * Swift 4.0+
 * iOS 9+
 
+## Getting Started Guide
+
+Please see our [Getting Started](https://github.com/plangrid/ReactiveLists/blob/master/Guides/Getting%20Started.md) guide to learn how to use ReactiveLists
+
+
+#### `SectionModel`
```
Comment (jessesquires) [inline README.md]: We probably don't want to dupe this stuff. I think we should keep the link above and have that be the single source of truth.

Comment (jessesquires) [reply README.md]: (and remove all this)

Comment (anayini) [reply README.md]: oops yeah I thought I got rid of that from `README`.  Will drop it

File: Guides/Getting Started.md (hunk 2 above)
Comment (anayini) [reply Guides/Getting Started.md]: Do you mean to not mix the text explaining the code with the example code?  Or do you just mean pulling up the `Glossary of Types` (soon to be renamed :) ) above this section with code?

**Comment by anayini** [comment]
https://github.com/plangrid/ReactiveLists/issues/11

**Comment by anayini** [comment]
Landing for now - we can rereview at EOD

**Pull request merged**